### ISSUE TITLE
Cargo parsing of before repackaged / after unpackaged items

### ISSUE THREAD

**Issue by Salvoxia** [molbal/abyss-tracker#14]
Hi,

there seems to be in issue with the cargo parser involving items changing their singleton state from before to after cargo.
Take an MTU for example. Before the run, the MTU might be repackaged (because it's brand new) and looks like this in cargo copy/paste:
```
Mobile Tractor Unit	1	Mobile Tractor Unit			100 m3	7.546.602,14 ISK
```
Notice quantity is 1 on this.

When it is no longer repackaged, it looks like this, the quantity is now gone:
```
Mobile Tractor Unit		Mobile Tractor Unit			100 m3	7.546.602,14 ISK
```

This causes Abyss Tracker to assume an item of that type was lost, but with quantity 0.
Example run where I know the MTU entered the Abyss repackaged and left it assembled in the cargo: 
https://abyss.eve-nt.uk/run/13368

Best Regards,
Salvoxia

**Comment by molbal** [comment]
Hi,

This is a difficult one as the parsing of the fields is "outsourced" to the Eve Praisal API.

I tried it both ways, by adding the packaged one to the before and the repackaged one to the after cargo and the other way around and I could not reproduce it.

I first tried it both ways my in local dev environment, then in the live system and I couldn't reproduce it anywhere.

Do you maybe have another test input that reproduces the bug more reliably?

Best Regards,
Veetor

**Comment by Salvoxia** [comment]
You're right, I can't reproduce it either. I'll close this issue and re-open when/if I ever notice it again and/or have more info.
Thanks for looking into this anyway!

**Comment by molbal** [comment]
Alright. I'll keep an open eye too

**Comment by stiez** [comment]
Hello,

I've got the described effect in the run https://abyss.eve-nt.uk/run/17968 with the following before and after cargo pastes. Just to be sure I've added a text file with both pastes because I don't trust the formatting of the comment section.

If you need further information I'll try to provide it if possible.

Best regards


[Missing_MTU.txt](https://github.com/molbal/abyss-tracker/files/4624563/Missing_MTU.txt)

Before:
Mobile Tractor Unit	1	Mobile Tractor Unit
Nanite Repair Paste	100	Nanite Repair Paste
Raging Exotic Filament	6	Abyssal Filaments
Scourge Fury Light Missile	5.051	Advanced Light Missile

After:
Decayed Stasis Webifier Mutaplasmid	1	Mutaplasmids
Entropic Radiation Sink I Blueprint		Entropic Radiation Sink Blueprint
Fierce Gamma Filament	1	Abyssal Filaments
Heavy Entropic Disintegrator I Blueprint		Precursor Weapon Blueprint
Mobile Tractor Unit		Mobile Tractor Unit
Nanite Repair Paste	100	Nanite Repair Paste
Raging Electrical Filament	2	Abyssal Filaments
Raging Exotic Filament	5	Abyssal Filaments
Scourge Fury Light Missile	4.891	Advanced Light Missile
Triglavian Survey Database	195	Triglavian Data
Zero-Point Condensate	138	Abyssal Materials

**Comment by molbal** [comment]
Hello,

The site uses Evepraisal to parse these they sometimes work sometimes do not.

Before: http://evepraisal.com/a/rqtdc/m4y9uggq022nn4tw
After: http://evepraisal.com/a/rqtds/ndidnr4cc5g3855h and sometimes http://evepraisal.com/a/rqtd4/qsllkw2bg5v0xfnv

I'm talking with the Eve workbench guys to use their Appraisal tool (Building one from scratch is out of scope for me) and that might be more reliable. We'll see :)

**Comment by molbal** [comment]
Published fix with 1.5.2.
